assets/js/blocks/cart-checkout/cart-i2/inner-blocks/filled-cart-block/frontend.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { SidebarLayout } from '@woocommerce/base-components/sidebar-layout';
+
+const FrontendBlock = ( {
+	children,
+}: {
+	children: JSX.Element;
+} ): JSX.Element => {
+	// @todo pass attributes to inner most blocks.
+	const hasDarkControls = false;
```

Comment:
I don't know how to pass attributes yet.

Reply:
We will use the context provider probably from top level. Can be handled later.